### PR TITLE
Assignment 3

### PR DESCRIPTION
**Pull request by dnisyndrum**
Completed. Added PrintInfo() functions for food and television and child classes for actor. I included Howard as well. You can't have the Big Bang Theory gang without Howard! Tests added for food, television, and the actors using the Speak() extension method.

### CODE REVIEW

**Comment by TatyanaHubbard** [comment]
Good work! I think you hit all of the possible tests and got the prompt right!

File: Inheritance/src/Inheritance/ActorMixins.cs [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+namespace Inheritance
+{
+    public static class ActorMixins
+    {
+        public static string Speak(this Actor actor) => actor switch
+        {
+            Penny penny => penny.Speak(),
+            Sheldon sheldon => sheldon.Speak(),
+            Howard howard => howard.Speak(),
+            Raj raj => raj.WomenPresent ? raj.Mumble() : raj.Speak(),
```
Comment (breaman) [inline Inheritance/src/Inheritance/ActorMixins.cs]: Nice implementation here for the two cases with Raj.

File: Inheritance/test/Inheritance.Tests/ActorTests.cs [hunk 2]
```diff
@@ -6,8 +6,37 @@ namespace Inheritance.Tests
     public class ActorTests
     {
         [TestMethod]
-        public void TestMethod1()
+        public void Actor_Sheldon_Speak()
         {
+            Actor sheldon = new Sheldon();
+            Assert.AreEqual(sheldon.Speak(), "AAHHH!!! OH GOD, NOT EUCLID AVENUE!!!");
```
Comment (breaman) [inline Inheritance/test/Inheritance.Tests/ActorTests.cs]: Not docking you for this, but just an fyi on the assertions, it is the expected value first and then the actual value, so you would have wanted to swap these parameters.

File: Inheritance/test/Inheritance.Tests/PrinterTests.cs [hunk 3]
```diff
@@ -7,14 +7,63 @@
 namespace Inheritance.Tests
 {
     [TestClass]
-    public class PersonPrinterTests
+    public class PrinterTests
```
Comment (breaman) [inline Inheritance/test/Inheritance.Tests/PrinterTests.cs]: Nice job on the unit tests, very thorough.

**Pull request closed without merging**